readme.md

```diff
@@ -18,41 +18,63 @@ Enable Accelerated Mobile Pages (AMP) on your WordPress site.
 
 Bring the speed and features of the open source [AMP project](https://www.ampproject.org/) to your site, the WordPress way.
 
-With the plugin active, all posts on your site will have AMP-compatible versions, accessible by appending `/amp/` to the end your post URLs. For example, if your post URL is `http://example.com/2016/01/01/amp-on/`, you can access the AMP version at `http://example.com/2016/01/01/amp-on/amp/`. If you do not have [pretty permalinks](https://codex.wordpress.org/Using_Permalinks#mod_rewrite:_.22Pretty_Permalinks.22) enabled, you can do the same thing by appending `?amp=1`, i.e. `http://example.com/?p=123&amp=1`
+Run AMP on your entire site, with your own theme. This means little or no custom development to get the same experience in AMP.
 
-Also, your pages and custom post types can have AMP versions. Simply check their boxes on the 'AMP Settings' page in `/wp-admin`.
+There can be challenges, like reimplementing JavaScript. But the plugin will let you know when it removed invalid nodes.
 
-Your entire site can render as "Native AMP" if your theme calls `add_theme_support( 'amp' )`. There will only be one version of each URL: the AMP version. There won't be separate URLs with `/amp` or `?amp` appended. See this [wiki page](https://github.com/Automattic/amp-wp/wiki/Adding-Theme-Support#native-amp) for details and restrictions.
+There's a UI to help you ensure that AMP URLs look and function the same as in non-AMP.
 
-Your theme can also use [Paired Mode](https://github.com/Automattic/amp-wp/wiki/Adding-Theme-Support#paired-mode), with your own custom templates for the AMP URLs.
+Use your own theme by selecting "Native" or "Paired" mode on the "AMP Settings" page. These have full support of commenting and widgets.
 
-"Native AMP" and "Paired Mode" add full support for commenting and widgets.
+"Native" mode will render your entire site as AMP. There will only be one version of each URL: the AMP version. There won't be separate URLs with `/amp` or `?amp` appended.
 
-If your theme doesn't support `'amp'`, this will use basic legacy post templates for AMP consumers like Google Search and Twitter. And when visiting the site, the AMP content won't normally appear without appending strings to the URL like `/amp` or `?amp`.
+"Paired" mode has an AMP URL and a non-AMP URL for the templates where AMP is enabled. For example, the non-AMP URL of `http://example.com/foo/` will have an AMP URL of `http://example.com/foo/?amp`.
+
+In both of these modes, you can select which templates should have AMP enabled. Choose a single template type like "Categories," a few templates, or your entire site.
+
+There's also a way for themes to set these modes by adding theme support of `'amp'`. Please see this document for more details.
```

Comment:
`this document` should link to the product site.

Reply:
It could be a link to the wiki for now.

Reply:
Good point. https://github.com/Automattic/amp-wp/pull/1561/commits/efec95c214172c00f659f26f7b43c32c48865cf2#diff-eb6b6c90251ab33cee784713c451e6d8R33 adds a link to that wiki page.